common/src/main/java/tds/common/configuration/RestTemplateConfiguration.java

```diff
@@ -1,14 +1,29 @@
 package tds.common.configuration;
 
-import org.springframework.boot.web.client.RestTemplateBuilder;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Import;
+import org.springframework.http.client.BufferingClientHttpRequestFactory;
+import org.springframework.http.client.SimpleClientHttpRequestFactory;
 import org.springframework.web.client.RestTemplate;
 
+import java.util.Arrays;
+
+import tds.common.web.interceptors.RestTemplateLoggingInterceptor;
+
 @Configuration
+@Import(JacksonObjectMapperConfiguration.class)
 public class RestTemplateConfiguration {
+
+    @Autowired
+    private ObjectMapper objectMapper;
+
     @Bean
-    public RestTemplate restTemplate(RestTemplateBuilder builder) {
-        return builder.build();
+    public RestTemplate restTemplate() {
```

Comment:
Should we be configuring our RestTemplate like [this](http://docs.spring.io/spring-boot/docs/current/reference/html/boot-features-restclient.html)?  Subtle difference but it seems to be the current way?

I think the builder will still allow you to do all you have below.